client/js/is-sum-formula.js

```diff
@@ -0,0 +1,40 @@
+const isSumFormula = function(string, numCols, numRows) {
```

Comment:
`isValidSumFormula` would be a better name, since you are doing some validation here.